mucgpt-frontend/src/components/ContextManagerDialog/ContextManagerDialog.module.css

```
@@ -0,0 +1,182 @@
+    display: flex;
+    flex-direction: column;
+    gap: 2px;
+    overflow: hidden;
+    min-width: 0;
+}
+
+.fileName {
+    font-weight: 600;
+    font-size: 0.9375rem;
+    color: var(--colorNeutralForeground1);
+    word-break: break-word;
```

Stylelint (17.6.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)